cfme/tests/intelligence/reports/test_crud.py

```diff
@@ -83,26 +85,48 @@ def test_custom_report_crud(custom_report_values, appliance):
 
 @pytest.mark.sauce
 @pytest.mark.tier(3)
+@pytest.mark.meta(automates=[BZ(1729882), BZ(1202412), BZ(1446052)])
 def test_reports_schedule_crud(schedule_data, appliance):
     """
-    Bugzilla:
-        1202412
-
     Polarion:
         assignee: pvala
         casecomponent: Reporting
         caseimportance: high
         initialEstimate: 1/16h
+
+    Bugzilla:
+        1202412
+        1446052
+        1729882
     """
+    # create
     schedule = appliance.collections.schedules.create(**schedule_data)
     view = schedule.create_view(ScheduleDetailsView)
```

Comment:
I know this is out of the scope of this PR, just curious, you know why are we creating view instead of navigating to it?

Reply:
Once the schedule is created, we are navigated to `ScheduleDetailsView`. We create that view so that we can assert flash messages on that page, if we re-navigate to that page, flash messages will disappear.